swarm/runtime/statsdb/ingestion.py

```
@@ -49,6 +62,73 @@ def _insert_raw_event(self, event: Dict[str, Any]) -> bool:
+                    event.get("agent_key"),
+                    json.dumps(event.get("payload", {})),
+                ]
+            )
+
+        with self._lock:
+            try:
+                self.connection.execute("BEGIN TRANSACTION")
+                self.connection.executemany(query, params_list)
+                self.connection.execute("COMMIT")
+                return True
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)